brackend/db/models.py

```diff
@@ -1,11 +1,21 @@
 """Module that defines/creates/holds ORMs for the database."""
-from brackend.util import BrackendException, DOCKER_POSTGRES_URL
-from sqlalchemy import Boolean, Column, ForeignKey, Integer, String, create_engine
+import enum
+from datetime import datetime
+
+from sqlalchemy import Boolean, Column, DateTime, Enum, ForeignKey, Integer, String, create_engine
 from sqlalchemy.orm import declarative_base, relationship
 
+from brackend.util import DOCKER_POSTGRES_URL, BrackendException
+
 Base = declarative_base()
 
 
+class ProgressEnum(enum.Enum):
+    not_started = 1
+    in_progress = 2
+    completed = 3
```

Comment:
also nit, if you import auto from enum you can do 
```suggestion
    not_started = auto()
    in_progress = auto()
    completed = auto()
```

but matter of preference